### ISSUE TITLE
Bookmarks are not saved

### ISSUE THREAD

**Issue by AJolly** [hassio-addons/addon-sqlite-web#319]
This doesn't seem to save bookmarks across browsers, which makes it much less useful.

**Comment by tjorim** [comment]
Correct, the bookmarks are stored in your browser's localStorage. This is by design and not something the add-on can control: https://github.com/coleifer/sqlite-web
